src/app/personside/infotabs/meldinger/traadvisning/Enkeltmelding.tsx

```diff
@@ -38,17 +35,35 @@ const SnakkebobleWrapper = styled.div`
     }
 `;
 
+const JournalforingEtikett = styled(EtikettLiten)`
+    color: ${theme.color.lenke};
+`;
+
+const JournalforingPanel = styled(EkspanderbartpanelBase)`
+    .ekspanderbartPanel__hode {
+        padding: 0.5rem 0 0 0;
+    }
+    .ekspanderbartPanel__innhold {
+        padding: 0.5rem 0 0 0;
+    }
+`;
+
 function journalfortMelding(melding: Melding) {
     const navn = melding.journalfortAv && saksbehandlerTekst(melding.journalfortAv);
     const dato = melding.journalfortDato && formaterDato(melding.journalfortDato);
     return `Journalført av ${navn} ${dato} på tema ${melding.journalfortTemanavn} med saksid ${melding.journalfortSaksid}`;
 }
 
 function Journalforing({ melding }: { melding: Melding }) {
+    const [open, setOpen] = useState(false);
     return melding.journalfortAv && melding.journalfortDato ? (
-        <JournalforingStyle>
-            <EtikettGrå>{journalfortMelding(melding)}</EtikettGrå>
-        </JournalforingStyle>
+        <JournalforingPanel
+            heading={<JournalforingEtikett>Meldingen er journalført</JournalforingEtikett>}
```

Comment:
Kunne man bare brukt `<Etikett type="suksess">` her eller blir den seende for stor ut?

Reply:
Det er også en mulighet. Jeg synes det blir litt for mye etiketter på hver av meldingene. Men kan ta en diskusjon på hva som er best for brukeren :)
<img width="687" alt="Screen Shot 2019-09-30 at 12 33 57 PM" src="https://user-images.githubusercontent.com/11627538/65871420-a4b1a200-e37e-11e9-9b90-7fe20778dd96.png">

Reply:
Tror kanskje det bare er jeg som ble forvirret av "etikett"-ordet. Det er ett kjent begrep fra designmanualen her på huset. Så om vi kan unngå å bruke det ordet på ting som ikke skal se ut som en etikett så blir jeg happy. :)

Reply:
Veldig enig i den! _JournalforingsLabel_ er nok bedre!